css/styles.css

```diff
@@ -1,322 +1,407 @@
-.clearfix:after {
-     visibility: hidden;
-     display: block;
-     font-size: 0;
-     content: " ";
-     clear: both;
-     height: 0;
-     }
-.clearfix { display: inline-block; }
-/* start commented backslash hack \*/
-* html .clearfix { height: 1%; }
-.clearfix { display: block; }
-/* close commented backslash hack */
-
-body {
-  background-image: url('../img/grass.png');
-  margin: 0;
-  padding: 0;
-  overflow: hidden;
-  font-family: sans-serif;
-  font-size: 55px;
-  font-weight: bold;
+.clearfix:after
```

Comment:
hat es einen bestimmten grund, weshalb die css formatierung geändert wurde ?

Reply:
Ist meine Formatierung in VS. Akzeptiere es oder stell ein PR

Reply:
oha rastet der direkt aus 🗡  ;D

Reply:
Haha Nein war nur eine Anmerkung... Ist nicht nur bei der Datei sondern auch in den js Files.

Sonst klebt alles nebeneinander und du hast keinen Plan mehr was da abgeht und die Fehlerquote steigt. :/

Reply:
Haha Nein war nur eine Anmerkung... Ist nicht nur bei der Datei sondern auch in den js Files.

Sonst klebt alles nebeneinander und du hast keinen Plan mehr was da abgeht und die Fehlerquote steigt. :/

Reply:
Ich warte immernoch auf deinen PR Mensur =D